client/my-sites/backup/backup-contents-page/file-browser/index.tsx

```diff
@@ -1,20 +1,26 @@
 import { useState } from '@wordpress/element';
 import { FunctionComponent } from 'react';
+import { useSelector } from 'calypso/state';
+import { getSelectedSiteId } from 'calypso/state/ui/selectors';
 import FileBrowserHeader from './file-browser-header';
 import FileBrowserNode from './file-browser-node';
 import { FileBrowserItem } from './types';
 
 interface FileBrowserProps {
 	rewindId: number;
 	showHeaderButtons?: boolean;
+	siteId?: number;
 }
 
 const FileBrowser: FunctionComponent< FileBrowserProps > = ( {
 	rewindId,
 	showHeaderButtons = true,
+	siteId: propSiteId,
```

Comment:
I think this renaming of the prop could be confusing with the declaration of the same prop below in [line 23](https://github.com/Automattic/wp-calypso/pull/104648/files#diff-acbcf8f2dbe74f4b9d3719c115129e4dedd87d0eff554d25913c34f878c9d87eR23). I think you can use `siteId` as is and then create a `calculatedSiteId` or `effectiveSiteId` that can be passed to the other inner components. Not a hard requirement though, it was just confusing to me at the beginning.

Reply:
I agree that it reads more clearly if we keep `siteId` in the prop and use another name later. Done in 6c5acad948b.